src/main/java/cooper/meetings/MeetingManager.java

```diff
@@ -1,21 +1,31 @@
 package cooper.meetings;
 
+import cooper.exceptions.DuplicateUsernameError;
 import cooper.ui.Ui;
 
+import java.util.ArrayList;
 import java.util.HashMap;
 
 public class MeetingManager {
-    private static HashMap<String, String> meetings = new HashMap<>(10);
+    private static HashMap<String, ArrayList<String>> meetings = new HashMap<>(10);
 
-    public static void addAvailability(String time, String name) {
-        meetings.put(time, name);
+    public static void addAvailability(String time, String name) throws DuplicateUsernameError {
+        if (!meetings.containsKey(time)) {
+            meetings.put(time, new ArrayList<>());
+        }
+
+        // check if the value is a duplicate
+        if (!meetings.get(time).contains(name)) {
+            meetings.get(time).add(name);
+        } else {
+            throw new DuplicateUsernameError();
+        }
     }
 
     public static void printAvailabilities() {
         Ui.printMeetingTableHeader();
         for (String key: meetings.keySet()) {
             Ui.showText(key + " | " + meetings.get(key));
```

Comment:
Do you think we can make a method in `Ui` to print out the `ArrayList` of names according to a certain format? (cause `meetings.get(key)` returns an `ArrayList` right?). 
If i'm not wrong, printing it now will just call the `toString()` method for `ArrayList`s.